format

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+find include libc src -name '*.h' -o -name '*.c' | xargs clang-format -i
```

Comment:
seems `libc` doesn't exist (yet?)

Reply:
yeah it will part of the next phase, which is making a PR for the headers (and only the headers), I think it's fine to keep it for now even if it's not there yet